cves/kernel/CVE-2016-3135.yml

```diff
@@ -360,8 +368,10 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: | 
+    The fix involves a small if statement that checks whether the size
```

Comment:
size of what?

Reply:
The size of the memory being allocated for an object. Nice catch & it has been updated.